lib/translator/common.cc

```diff
@@ -199,4 +199,23 @@ const char* ScsiOpcodeToString(scsi::OpCode opcode) {
   }
 }
 
+bool FillSenseBuffer(Span<uint8_t> sense_buffer, ScsiStatus& scsi_status) {
```

Comment:
```suggestion
bool FillSenseBuffer(Span<uint8_t> sense_buffer, const ScsiStatus& scsi_status) {
```
Use const references whenever possible. This makes it clearer what the intention of the function is and allows it to be used in more contexts.